developers/wcs/embeddings/models.md

```diff
@@ -0,0 +1,172 @@
+
+---
+-->
+### `Snowflake/snowflake-arctic-embed-m-v1.5` {#snowflake-arctic-embed-m-v1.5}
+
+- A 109M parameter, 768-dimensional model for enterprise retrieval tasks in English.
+- Trained with Matryoshka Representation Learning to allow vector truncation with minimal loss.
+- Quantization-friendly: Using scalar quantization and 256 dimensions provides 99% of unquantized, full-precision performance.
+- Read more at the [Snowflake blog](https://www.snowflake.com/engineering-blog/arctic-embed-m-v1-5-enterprise-retrieval/), and the Hugging Face [model card](https://huggingface.co/Snowflake/snowflake-arctic-embed-m-v1.5)
+- Allowable `dimensions`: 768 (default), 256
+
+:::info Input truncation
+Currently, input exceeding the model's context windows is truncated from the right (i.e. the end of the input).
+:::
+
+#### Parameters
+
+- `model` (optional): The name of the model to use for embedding generation.
+- `dimensions` (optional): The number of dimensions to use for the generated embeddings. Only available for models that support Matryoshka Representation Learning.
+- `base_url` (optional): The base URL for the Weaviate Embeddings service. (Not required in most cases.)
+
+The following examples show how to configure Weaviate Embeddings-specific options.
+
+<Tabs groupId="languages">
+  <TabItem value="py" label="Python API v4">
+    <FilteredTextBlock
+      text={PyCode}
+      startMarker="# START SnowflakeArcticEmbedMV15"
+      endMarker="# END SnowflakeArcticEmbedMV15"
+      language="py"
+    />
+  </TabItem>
+
+  <TabItem value="js" label="JS/TS API v3">
+    <FilteredTextBlock
+      text={TSCode}
+      startMarker="// START SnowflakeArcticEmbedMV15"
+      endMarker="// END SnowflakeArcticEmbedMV15"
+      language="ts"
+    />
+  </TabItem>
+
+  <TabItem value="go" label="Go">
+    <FilteredTextBlock
+      text={GoCode}
+      startMarker="// START SnowflakeArcticEmbedMV15"
+      endMarker="// END SnowflakeArcticEmbedMV15"
+      language="goraw"
+    />
+  </TabItem>
+
+  <TabItem value="java" label="Java">
+    <FilteredTextBlock
+      text={JavaCode}
+      startMarker="// START SnowflakeArcticEmbedMV15"
+      endMarker="// END SnowflakeArcticEmbedMV15"
+      language="java"
+    />
+  </TabItem>
+
+</Tabs>
+
+---
+
+### `Snowflake/snowflake-arctic-embed-l-v2.0` (default) {#snowflake-arctic-embed-l-v2.0}
+
+- A 568M parameter, 1024-dimensional model for multilingual enterprise retrieval tasks.
+- Trained with Matryoshka Representation Learning to allow vector truncation with minimal loss.
+- Quantization-friendly: Using scalar quantization and 256 dimensions provides 99% of unquantized, full-precision performance.
+- Read more at the [Snowflake blog](https://huggingface.co/Snowflake/snowflake-arctic-embed-l-v2.0), and the Hugging Face [model card](https://huggingface.co/Snowflake/snowflake-arctic-embed-l-v2.0)
+- Allowable `dimensions`: 768 (default), 256
```

Comment:
I think the default is 1024.

Reply:
ref: https://github.com/weaviate/weaviate/blob/6bd3288a6b988e1dcad705c9d760a93c2551a365/modules/text2vec-weaviate/ent/class_settings.go#L44

Reply:
Thank you, I missed this when copy-pasting.